php/tests/WebhookTest.php

```diff
@@ -52,6 +52,18 @@ public function testInvalidSignatureThrowsException()
         $wh->verify($testPayload->payload, $testPayload->header);
     }
 
+    public function testBadlyFormattedSignatureThrowsException()
+    {
+        $this->expectException(\Svix\Exception\WebhookVerificationException::class);
+        $this->expectExceptionMessage("No matching signature found");
+
+        $testPayload = new TestPayload(time());
+        $testPayload->header['svix-signature'] = 'BAD_SIG_NA_TURE';
```

Comment:
```suggestion
        $testPayload->header['svix-signature'] = 'BAD_SIG_NATURE';
```
Just doing this to make the typo checker happy.